CHANGELOG.adoc

```diff
@@ -14,6 +14,7 @@
 
 - [ADR-158] Support delegating representation event handling
 - [ADR-159] Create event processor only using the representation id
+- [ADR-161] Add support for actions in tree item menu
```

Comment:
Since Github doesn't allow me to comment commit messages I'll write it here but don't add prefixes like "ADR:" in the commit message. We already have a tag for task related to the documentation `[doc]`

Reply:
Changed